src/code-editor/ace-editor.scss

```diff
@@ -12,10 +12,10 @@
 
 .code-editor-refresh :global .ace_editor {
   .ace_gutter {
-    border-top-left-radius: calc(awsui.$border-radius-code-editor - awsui.$border-item-width);
+    border-start-start-radius: calc(awsui.$border-radius-code-editor - awsui.$border-item-width);
```

Comment:
Since these are overriding built-in ace styles, how does it work? Do logical properties override non-logical?

Reply:
Logical and physical properties are effectively setting the same value and it just comes down to selector specificity.

Reply:
It would be no different than having two selectors set different values for the same property. The highest specificity selector value wins. There is no inherent hierarchy to physical vs logical properties to my knowledge.

Reply:
👍